### PR TITLE
LPS-115747 Use CKEDITOR's enhanced image plugin

### PR DESCRIPTION
**Pull request by julien**
In order to complete [LPS-115747](https://issues.liferay.com/browse/LPS-115747), 
we've enabled CKEDITOR's [Enhanced Image](https://ckeditor.com/cke4/addon/image2) plugin (also referred to as `image2`) 
which allows center aligning images - as well as resizing them

This is what it looks like in Knowledge Base articles

Steps to reproduce:

- Navigate to Content & Data > Knowledge Base
- Click on the "+" button and choose "Basic Article"
- In the content editor, click on the image icon to choose an image

Double clicking on the image opens a dialog with the image's properties, that lets you align the image to the left, to the center or to the right. You can also use the alignment buttons in the editor's toolbar to change the image's alignment.

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by julien** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:bundle

**Comment by jbalsas** [comment]
Wondering why this is tested in `Blogs` which should be showing an `AlloyEditor` instance rather than a CKEditor and not in every other place where we do have CKEditor by default.

There are no "Steps to reproduce" in the description, so having a hard time wrapping my head around this.

**Comment by julien** [comment]
@jbalsas I updated the description and the screenshot.

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_editor_ckeditor.scss [hunk 1]
```diff
@@ -39,14 +39,3 @@
 		height: 15px;
 	}
 }
-
-// ---------- CKEditor image resizer ----------
```
Comment (carloslancha) [inline modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_editor_ckeditor.scss:43]: Removing code?

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2FB54TA1aiS9WSI%2Fgiphy.gif&token=approve" data-gitmeme-token="approve"></a>

**Comment by carloslancha** [comment]
@julien this pr will probably conflict with https://github.com/liferay-frontend/liferay-portal/pull/240

**Comment by julien** [comment]
@carloslancha thanks, since #240 was merged I'll check that

**Comment by julien** [comment]
@carloslancha it doesn't conflict with `master` so I don't understand why there would be a conflict - can you help me understand?

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_editor_ckeditor.scss (hunk 1 above)
Comment (jbalsas) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_editor_ckeditor.scss:43]: applause

**Comment by jbalsas** [comment]
If it conflicts we'll know soon enough... going to ship it to get it out of the way and make room for more incoming PRs!

**Comment by jbalsas** [comment]
ci:forward

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_editor_ckeditor.scss (hunk 1 above)
Comment (julien) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_editor_ckeditor.scss:43]: :rofl:

**Pull request closed without merging**